lib/cisco_node_utils/bgp.rb

```diff
@@ -313,6 +313,54 @@ def default_confederation_id
       config_get_default('bgp', 'confederation_id')
     end
 
+    #
+    # disable-policy-batching (Getter/Setter/Default)
+    #
+    def disable_policy_batching
+      config_get('bgp', 'disable_policy_batching', @get_args)
+    end
+
+    def disable_policy_batching=(enable)
+      @set_args[:state] = (enable ? '' : 'no')
+      config_set('bgp', 'disable_policy_batching', @set_args)
+      set_args_keys_default
+    end
+
+    def default_disable_policy_batching
+      config_get_default('bgp', 'disable_policy_batching')
+    end
+
+    # disable-policy-batching
+    # disable-policy-batching <afi> prefix-list <prefix_list>
+    #
+    def disable_policy_batching_prefix
```

Comment:
Does this property only apply to ipv4/ipv6, or are we going to eventually proliferate into `disable_policy_batching_l2vpn`, `disable_policy_batching_vpnv4`, etc?